packages/react-dom/src/shared/ensureCorrectIsomorphicReactVersion.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import reactDOMPackageVersion from 'shared/ReactVersion';
+import * as IsomorphicReactPackage from 'react';
```

Comment:
Since we compile to CJS this doesn't matter much but it might be nice to `import {version} from "react"` so that it doesn't pull in every export and disables dead export elimination.

Reply:
I've been doing it this way based on this comment. Not sure if it still applies:

https://github.com/facebook/react/blob/e8b016ccc6da25787f3821c463644301412d0f97/packages/react-reconciler/src/Scheduler.js#L10-L15